cli/js/tests/README.md

```diff
@@ -37,11 +37,32 @@ ways:
 - sanitization of async ops - ensuring that tests don't leak async ops by
   ensuring that all started async ops are done before test finishes
 
-`unit_test_runner.ts` is main script used to run unit tests.
+## Running tests
+
+`unit_test_runner.ts` is the main script used to run unit tests.
 
 Runner discoveres required permissions combinations by loading
 `cli/js/tests/unit_tests.ts` and going through all registered instances of
-`unitTest`. For each discovered permission combination a new Deno process is
-created with respective `--allow-*` flags which loads
-`cli/js/tests/unit_tests.ts` and executes all `unitTest` that match runtime
-permissions.
+`unitTest`.
+
+There are three ways to run `unit_test_runner.ts`:
+
+- run tests matching current process permissions
+
+  // run tests that don't require any permissions deno unit_test_runner.ts
+
+  // run tests with "net" permission deno --allow-net unit_test_runner.ts
+
+  deno --allow-net --allow-read unit_test_runner.ts
```

Comment:
The commands would be more useful from the root of the project directory and using target/debug/deno explicitly. Then I could copy and paste :)

Reply:
Yeah, just thought about it!